pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.609.1</version>
```

Comment:
Agreed the comments by @jglick do make sense and I'll do something about them as soon as I get the time.

The "Plugin Compatibility with Workflow"-documentation (https://github.com/jenkinsci/workflow-plugin/blob/master/COMPATIBILITY.md#build-wrappers-1) states that "To add support for a BuildWrapper, depend on Jenkins 1.599+ (typically 1.609.1), and implement SimpleBuildWrapper, following the guidelines in its Javadoc." so thats the reason for depending on 1.609.1. I guess it is a fairly strong reason.